firmware/library/L1_Drivers/ssp.hpp

```diff
@@ -0,0 +1,206 @@
+/**
+ ** Provide How-to for driver

```

Comment:
Please place a TODO here.
The todo should look like this:
`// TODO(#1241) Provide How-to for driver`

Where #1241 is the issue number

Also, only use // for comments including multiline comments

Reply:
This should also be just 1 line for now.

Reply:
Is the Issue number an arbitrary number, or is #1241 a specific issue # that I'm not seeing right now?

Reply:
Its whatever git assigns the number to be.